src/utils/util.js

```diff
@@ -116,6 +130,15 @@ function getDisplayBoundaries() {
   });
 }
 
+// next two functions are defined to clarify intent
+function testUrlsWithSubpath(url1, url2) {
```

Comment:
What are we testing about these URLs?

Reply:
I'll think of a better name, it might confuse people